engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -210,7 +208,7 @@ public void initialize() {
 
     var preinit = environment.getOptions().get(RuntimeOptions.PREINITIALIZE_KEY);
     if (preinit != null && preinit.length() > 0) {
-      var epb = environment.getInternalLanguages().get(ForeignLanguage.ID);
+      var epb = environment.getInternalLanguages().get("epb");
```

Comment:
Do no hardcode "epb". Use `EpbLanguage.ID`. In `build.sbt`, `runtime` project should depend on `runtime-language-epb`, right?

Reply:
No. These two projects providing their languages shall be independent and loosely coupled. _The harm is limitted_ to four occurrences https://github.com/enso-org/enso/pull/7882#discussion_r1426623105 and tests exist to ensure it all works together.